src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+import { Post } from '../../types/Post';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo
+        post={post}
+        key={post.userId}
```

Comment:
Why do you use userId as a key? 
A few posts can be created by the same user, so they would have the same userId.
I think it's better to use post.id as a key as it's unique for each post.

Reply:
Yea, I'm agree! Ididn't teake this moment into account!Thank's)